test/src/ui/js/test_Game.js

```diff
@@ -1548,6 +1548,168 @@ test("test_Game.gamePlayerDice_warrior", function(assert) {
   });
 });
 
+test("test_Game.isDieClickable", function(assert) {
+  var player = '';
```

Comment:
Nit: IMO these tests would be more readable if you didn't keep redefining player as a variable, but just used an inline string every time you invoked `isDieClickable()` --- i know that's picky, it's just that it would make it easier to scan the tests at a glance and see the settings.